### PR TITLE
Add PHP 7.3.0RC2

### PR DESCRIPTION
**Pull request by desrosj**
[PHP 7.3.0RC2 was released on September 28th](https://secure.php.net/archive/2018.php#id2018-09-28-1). This PR adds support for that version.

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Update this when a new stable version comes around
-STABLE_DEFINITIONS="5.3.29 5.4.45 5.5.38 5.6.38 7.0.32 7.1.22 7.2.10 7.3.0RC1"
+STABLE_DEFINITIONS="5.3.29 5.4.45 5.5.38 5.6.38 7.0.32 7.1.22 7.2.10 7.3.0RC3"
```
Comment (BanzaiMan) [inline run-tests.sh]: You mean RC2, right?

Comment (desrosj) [reply run-tests.sh]: Oops, I'll fix that right now.

Comment (desrosj) [reply run-tests.sh]: All set. Thanks for catching, @BanzaiMan!

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**